### PR TITLE
(EWL-968): Handle Fullscreen

### PR DESCRIPTION
**Pull request by saad-kh**
# Handle fullscreen for ETP

Full screen is completely handled by the PlayerSkin as ETP will behave a simple canvas that gets resized without caring of being full screen or not.

### CODE REVIEW

**Comment by JulienKode** [comment]
How did you test it ? 
When I click on fullscreen it do nothing 

Which stories I need to use

**Comment by saad-kh** [comment]
did you npm install to have the last version of the player?

File: src/components/JWPlayerETP/PlayerSkin/useFullscreen.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    return () => {
+      document.removeEventListener('fullscreenchange', handleFullscreenChange);
+    };
+  }, [isFullscreen, containerElement]);
+
+  const onFullscreenChange = useCallback(() => {
+    if (isFullscreen) {
+      document.exitFullscreen();
+    } else if (containerElement) {
+      containerElement.requestFullscreen();
```
Comment (ra-phael) [inline src/components/JWPlayerETP/PlayerSkin/useFullscreen.js]: Will that work on all browsers?  Else we might use something like https://github.com/sindresorhus/screenfull.js/

Comment (saad-kh) [reply src/components/JWPlayerETP/PlayerSkin/useFullscreen.js]: fixed in https://github.com/EurosportDigital/web-toolkit/pull/299/commits/204f4b51ef2e598f2c5788403e396bf9d929a1ae

**Comment by JulienKode** [comment]
Good job @saad-kh ✅

It works pretty well on Safari Mac 
but I got that error on safari iOS 

67667724_2683648141862287_9054268041856024576_n

File: src/components/JWPlayerETP/PlayerSkin/useFullscreen.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    screenfull.on('change', handleFullscreenChange);
+    return () => {
+      screenfull.off('change', handleFullscreenChange);
+    };
+  }, [isFullscreen, containerElement]);
+
+  const onFullscreenChange = useCallback(() => {
+    if (isFullscreen) {
+      screenfull.exit();
```
Comment (JulienKode) [inline src/components/JWPlayerETP/PlayerSkin/useFullscreen.js]: Do we test that path ?

**Comment by saad-kh** [comment]
@JulienKode I added the test for when isFullscreen is true :)
Also iOS safari doesn't support requestFullscreen https://caniuse.com/#feat=fullscreen
for iPhones only iPad.
So I made sure we check that the feature is available before running the screenfull api. So for iOS safari on iPhones there will be no error, but the fullscreen won't work as it's not supported.
I'll let you validate and merge :)

**Comment by JulienKode** [comment]
## Checklist

- Browser compatibility checked:
  - [x] Safari
  - [x] Chrome
  - [x] Firefox
- [x] MR title respects the format: `(EWL-XXX): description`
- [x] Move Jira ticket into PR
- [x] Check with another developers that features sounds good

## Design

|      Browsers      |      iOS / Mac     |
| :--------------------: | :--------------------: |
|  | Screenshot 2019-08-06 at 17 06 25 |

## Description

**REPLACE ME** : help the reviewer here by indicating the steps you followed.

## Test Plan

**REPLACE ME** : help the reviewer here by indicating what did you test

**Pull request merged**